packages/devtools_app/lib/src/screens/memory/panes/chart/chart_pane.dart

```diff
@@ -590,7 +590,7 @@ class _MemoryChartPaneState extends State<MemoryChartPane>
     } else if (event[eventName] == devToolsEvent &&
         event.containsKey(customEvent)) {
       final custom = event[customEvent] as Map<dynamic, dynamic>;
-      final data = custom[customEventData];
+      final data = custom[customEventData] as Map<dynamic, dynamic>;
```

Comment:
should we also be switching `dynamic` out here and above for `Object?`

Reply:
Good catch, done.